monarchmoney/monarchmoney.py

```diff
@@ -209,6 +209,100 @@ async def get_accounts(self) -> Dict[str, Any]:
             graphql_query=query,
         )
 
+    async def get_account_type_options(self) -> Dict[str, Any]:
+        """
+        Retrieves a list of available account types and their subtypes.
+        """
+        query = gql(
+            """
+            query GetAccountTypeOptions {
+                accountTypeOptions {
+                    type {
+                        name
+                        display
+                        group
+                        possibleSubtypes {
+                            display
+                            name
+                            __typename
+                        }
+                        __typename
+                    }
+                    subtype {
+                        name
+                        display
+                        __typename
+                    }
+                    __typename
+                }
+            }
+        """
+        )
+        return await self.gql_call(
+            operation="GetAccountTypeOptions",
+            graphql_query=query,
+        )
+
+    async def create_manual_account(
+        self,
+        account_type: str,
+        account_sub_type: str,
+        is_in_net_worth: bool,
+        account_name: str,
+        account_balance: float = 0,
+    ) -> Dict[str, Any]:
+        """
+        Creates a new manual account
+
+        :param account_type: The string of account group type (i.e. loan, other_liability, other_asset, etc)
+        :param account_sub_type: The string sub type of the account (i.e. auto, commercial, mortgage, line_of_credit, etc)
+        :param is_in_net_worth: A boolean if the account should be considered in the net worth calculation
+        :param account_name: The string of the account name
+        :param display_balance: a float of the amount of the account balance when the account is created
+        """
+        query = gql(
+            """
+            mutation Web_CreateManualAccount($input: CreateManualAccountMutationInput!) {
+                createManualAccount(input: $input) {
+                    account {
+                        id
+                        __typename
+                    }
+                    errors {
+                        ...PayloadErrorFields
+                        __typename
+                    }
+                __typename
+               }
+            }
+            fragment PayloadErrorFields on PayloadError {
+                fieldErrors {
+                    field
+                    messages
+                    __typename
+                }
+                message
+                code
+                __typename
+            }
+            """
+        )
+        variables = {
+            "input": {
+                "type": account_type,
+                "subtype": account_sub_type,
```

Comment:
What happens if these are invalid?  Should the code pull the types implicitly and check them for folks?